### ISSUE TITLE
Use Progress.Circle in Expo

### ISSUE THREAD

**Issue by muhaimincs** [oblador/react-native-progress#115]

**Comment by terryBaz** [comment]
is it possible to use this library with **Expo** ? without injecting native code !

**Comment by muhaimincs** [comment]
I thought there is a room for without inject it

**Comment by bronydell** [comment]
Yes, it's possible because [ART is included in expo ](https://docs.expo.io/versions/latest/sdk/art)

**Comment by morenoh149** [comment]
anyone care to share a snack or minimal example? stumped on this

**Comment by giacomocerquone** [comment]
@oblador this can be closed, rn progress doesn't support expo as of now since it's based on ART and expo removed it
